web-server/views/users/profile.jade

```diff
@@ -0,0 +1,25 @@
+extends ../layout
+
+block content
+
+  .row
+    .col-md-4
+    .col-md-4.profile-header
+      h1= user.name
+      p.profile-links
+        a(href="#{user.profileUrl}" target="_blank") @#{user.handle}
+        a(href="https://echo.learnersguild.org/users/#{user.handle}" target="_blank") Echo
```

Comment:
some of these urls are already on the user object. Added from here https://github.com/GuildCrafts/curriculum/blob/backoffice-dashboard/backoffice/idm/index.js#L82

maybe we should add this as `user.echoURL` ?

Reply:
![image](https://user-images.githubusercontent.com/13281696/30564688-a3b1170c-9c7a-11e7-966c-2cc4a99cc4d5.png)

This seems to be all that is on the user object.